src/Subscriptions/SubscriptionPurchase.php

```diff
@@ -168,115 +167,115 @@ public static function fromResponseBody(array $responseBody): self
     }
 
     /**
-     * @return string
+     * @return string | null
```

Comment:
Please remove the not needed spacing:

Instead of `@return string | null`

Use `@return string|null`